Tasks/Mardoyan/Task5/Task5_5.py

```diff
@@ -0,0 +1,46 @@
+#Сразу извиняюсь)Знаю,что написано бредово,но я очень старался!Потом рад буду выслушать твои советы)
+
+user_input = input('Please input ordered list of numbers without duplicates tgrought "Space"\n')
+#Превращаю инпут в Int
+user_input = user_input.split(' ')
+data = []
+for i in user_input:
+    data.append(int(i))
+
+
+def range_func(a):
+    c = 0
+    d = [] #Список со списками(разделил каждый range)
+    z = [] #Список для каждых последующих эл.
+    g = [] #С диапазонами
```

Comment:
this structure can be reduced for example to a single list with all potential symbols (like ["1","-","3",",","4","-","9"]) that should be joined with an empty str or with edge numbers only (like ["1","-3","4","-9"]) where the right edge is stored along with a dash, should be joined with an empty string as well

the solution itself would be simplified accordingly